src/Table/Table.tsx

```diff
@@ -1229,9 +1229,9 @@ const Table: React.FunctionComponent<TableProps> = React.memo(
             const updatedColumns: Array<TableHeader> = props.columns.map((column: TableHeader) => {
                 return {
                     ...column,
-                    isSorted: false,
+                    isSorted: column.isSorted || false,
                     canSort: column.canSort !== undefined ? column.canSort : !!props.sortProps ? true : false,
-                    isSortedDesc: false,
+                    isSortedDesc: column.isSortedDesc || false,
```

Comment:
This doesn't seem right.

When columns are changed you shouldn't just return `isSorted` as `true` when the value is `true`. The value being of the `updatedColumns` are not sorted, it would be an invalid flag, wouldn't it?

Reply:
No, its a server side sorting and hence the data will be sorted by the server. The table will simply renders the data and will not sort the data.